src/creature.cpp

```diff
@@ -947,7 +949,7 @@ void Creature::getPathSearchParams(const Creature*, FindPathParams& fpp) const
 {
 	fpp.fullPathSearch = !hasFollowPath;
 	fpp.clearSight = true;
-	fpp.maxSearchDist = 12;
+	fpp.maxSearchDist = Map::maxViewportX + Map::maxViewportY;
```

Comment:
I like this! It is good and doesn't need changed, but I think IMO it would be better if it was +1 over the viewport max, as this would allow monsters to start being aggro'd just one square off of screen, which is how RL cibia is, but it's not really that important. Others may also prefer it to only be the size of the viewport.

Reply:
I was trying to decide here because in RL it is possible for monsters to path further than maxViewportX + 1. For instance if you are on the other side of a wall and right next to the monster on the other side. I am actually working on getting a character leveled up on RL to see exactly how they handle this exact situation. I do know monsters can path further, but if they do go to far away while trying to path to you they will stop pathing to you. Which is the effect this change has created.

Reply:
solution:
	fpp.maxSearchDist = (Map::maxViewportX + Map::maxViewportY) / 2;

Reply:
```
// Allow paths to be drawn a little further like rl tibia
fpp.maxSearchDist = (Map::maxViewportX * Map::maxViewportY) / 2;
```

Reply:
This would evaluate to like 60 something sqm which is a bit too much :D. Right now its 22 which I think it pretty adequate. Keep in mind most of the time if the distance ever reaches past like 10 sqm's it will stop as soon as the monster leaves the screen so it really isn't that big of a deal. I really think X+Y is the most logical and allows for people to change their viewports with minimal change to the range. X + 1 is a little to low based on paths I have seen cibia calculate but again once the monster leaves the screen + 2sqm it just stops trying to path.

Reply:
Sorry. It should have been divided by 4 and the reason why is (166/8)+1 = 22